### PR TITLE
[CAP 35] Implement ingestion for Clawback operations

### PR DESCRIPTION
**Pull request by 2opremio**
Depends on #3349 

Part of #3348

### CODE REVIEW

**Review by leighmcculloch — APPROVED** [review]

File: xdr/Stellar-ledger-entries.x [hunk 1]
```diff
@@ -356,6 +396,8 @@ struct ClaimableBalanceEntry
     {
     case 0:
         void;
+    case 1:
+        ClaimableBalanceEntryExtensionV1 v1;
```
Comment (bartekn) [inline xdr/Stellar-ledger-entries.x:400]: Looks like we need to update state verifier to support new fields.

**Review by bartekn — APPROVED** [review]

File: protocols/horizon/operations/main.go [hunk 2]
```diff
@@ -263,6 +265,22 @@ type RevokeSponsorship struct {
...
+	base.Asset
+	From   string `json:"from"`
+	Amount string `json:"amount"`
+}
+
+// Clawback is the json resource representing a single operation whose type is
+// Clawback.
+type ClawbackClaimableBalance struct {
+	Base
+	ClaimableBalanceID *string `json:"claimable_balance_id,omitempty"`
```
[the remark this answers is not in the document]
Comment (2opremio) [reply protocols/horizon/operations/main.go:281]: I just followed the same pattern as in the other claimable balance ops

File: xdr/Stellar-ledger-entries.x (hunk 1 above)
Comment (2opremio) [reply xdr/Stellar-ledger-entries.x:400]: Ah, good point, I will add it to the issue.

File: protocols/horizon/operations/main.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (bartekn) [reply protocols/horizon/operations/main.go:281]: Hmm, I wonder why we made it this way then.

**Pull request merged**